src/api/addressApi.ts

```diff
@@ -18,14 +18,43 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 
 import { ExplorerProvider } from '@alephium/web3'
 
-import { AddressDataResult, AddressHash, AddressTransactionsResult } from '@/types/addresses'
+import {
+  AddressAssetsResult,
+  AddressBalanceResult,
+  AddressHash,
+  AddressTransactionNumberResult,
+  AddressTransactionsResult
+} from '@/types/addresses'
 
-export const fetchAddressData = async (
+export const fetchAddressBalance = async (
   client: ExplorerProvider,
   addressHash: AddressHash
-): Promise<AddressDataResult> => {
+): Promise<AddressBalanceResult> => {
   const balances = await client.addresses.getAddressesAddressBalance(addressHash)
+
+  return {
+    addressHash,
+    balance: balances.balance,
+    lockedBalance: balances.lockedBalance
+  }
```

Comment:
very nit:
```suggestion
  return {
    addressHash,
    ...balances
  }
```

On second thought, for something as simple as an API call, is creating a method in the api directory really needed? :/ In this particular case, we are simply wrapping the API call.

Reply:
Same for below.

Reply:
Done